precompiles/xcm/src/lib.rs

```diff
@@ -188,21 +250,24 @@ where
             .into();
 
         // Build call with origin.
-        let origin = Some(R::AddressMapping::into_account_id(handle.context().caller)).into();
-        let call = pallet_xcm::Call::<R>::reserve_withdraw_assets {
+        let origin = Some(Runtime::AddressMapping::into_account_id(
+            handle.context().caller,
+        ))
+        .into();
+        let call = pallet_xcm::Call::<Runtime>::reserve_withdraw_assets {
```

Comment:
What I'd like to understand is what is the plan with `pallet-xcm` calls?

Some calls utilize `xtoken` calls, while some utilize `pallet-xcm` calls.

Reply:
We are still using the `pallet-xcm` calls for function that are in XCM interfaces and using `xtoken` calls for the function under XTOKENS interface. 
Although we can change most of the calls to use `xtokens` but I don't think there is an aching need for that. We will still have to use `pallet-xcm` calls for some functions like `send_xcm`.

Reply:
make sense, I will make a follow up PR for this.